mapentity/static/mapentity/mapentity.js

```diff
@@ -51,5 +52,22 @@ $(document).ready(function (e) {
         $.extend(data, context);
         $(window).trigger('entity:map', [data]);
         $(window).trigger('entity:map:' + context.viewname, [data]);
+
+        // Split
+        $("#panelleft").resizable({
+            handleSelector: ".splitter",
+            resizeHeight: false,
+            onDragEnd: function (e, $el, opt) {
+                window.maps[0].invalidateSize();
+            }
+        });
+        $(".details-panel").resizable({
+            handleSelector: ".splitter",
+            resizeHeight: false,
+            onDragEnd: function (e, $el, opt) {
+                window.maps[0].invalidateSize();
+            }
+        });
```

Comment:
You can DRY by setting resizable options ⬇️ 
```suggestion
        var resizableOptions = {
            handleSelector: ".splitter",
            resizeHeight: false,
            onDragEnd: function (e, $el, opt) {
                window.maps[0].invalidateSize();
            }
        }
        $("#panelleft").resizable(resizableOptions);
        $(".details-panel").resizable(resizableOptions);
```

I don't remember the jQuery engine selector well, maybe you can optimize like this: `$("#panelleft, .details-panel").resizable(resizableOptions);`

Reply:
I try this, thanks